### PR TITLE
My new branch

### PR DESCRIPTION
**Pull request by mwmentor**
20210827: I have made various updates to the Account Lookup & Discovery, Quotes & Agreements, Settlements & Transfers BCs.

### CODE REVIEW

File: docs/refarch/boundedContexts/thirdPartyApi/index.md [hunk 1]
```diff
@@ -1,22 +1,228 @@
...
+The following common terms are used in this BC:
 
-| Term | Description |
-|---|---|
-| Term1 | Description1 |
+| Term | Description  |
+| ---- | ------------ |
+| PISP | Payment Initiation Service Provider |
+| DFSP | Digital Financial Service Provider (Bank |
+| User | DFSP/PISP client (as indicated) |
```
Comment (mdebarros) [inline docs/refarch/boundedContexts/thirdPartyApi/index.md:13]: This is also generally known as a `Party` in Mojaloop.

I assume `User` is specific to 3PApi?

**Review by mdebarros — CHANGES_REQUESTED** [review]
Minor comment for your feedback

File: docs/refarch/boundedContexts/thirdPartyApi/index.md (hunk 1 above)
Comment (mdebarros) [reply docs/refarch/boundedContexts/thirdPartyApi/index.md:13]: @lewisdaly can you provide your thoughts on this?

Comment (mwmentor) [reply docs/refarch/boundedContexts/thirdPartyApi/index.md:13]: Perhaps it should be labelled "PISP user" TBC for now.  I have asked Lewis for his guidance.  Thanks

Comment (lewisdaly) [reply docs/refarch/boundedContexts/thirdPartyApi/index.md:13]: Yeah I think this is something like the common user between a PISP and DFSP. I think we used the term user and pisp user interchangeably.

**Review by mdebarros — APPROVED** [review]

**Pull request merged**